src/controllers/auth.js

```diff
@@ -209,9 +210,42 @@ async function signOut(req, res) {
     }
 }
 
+async function saveUserToTokenAndCookie(req, res) {
```

Comment:
I think it is more clear if we named the function saveGoogleUser, it will help when having another authentications like twitter to recognize functions easily!

Reply:
Tamam.